docs/presentations/articles/OS_LLMs_Robert.md

```diff
@@ -0,0 +1,278 @@
+<span style="color:red; font-size: 40px; font-weight: bold;">Open-Source LLMs</span>
+
+#### An Overview of Open Source LLMs: What Open Source Alternatives Exist to OpenAI's GPT-3.5 and How Can I Utilize Them?
+
+Autor: Karoly Robert Hegyi
+Datum: 21.12.2023
+
+Author: Karoly Robert Hegyi
+Date: 21.12.2023
+
+Open Source Large Language Models (OSLLMs) are powerful, open-source AI models for natural language processing tasks. Unlike proprietary models, they are transparent, facilitate collaboration, and foster innovation in applications such as translation, text generation, and chatbots.
+
+## Availability of Large Language Models
+
+They are distinguished into:
+
+| Proprietary LLMs                         | Open Source LLMs                          |
+| ---------------------------------------- | ----------------------------------------- |
+| Owned by a company that controls its use | Free of charge and accessible to everyone |
+| License fees required.                   | Free commercial use permitted.            |
+
+Various licenses exist for open-source software, with the majority adhering to the Apache 2.0 standard.
+
+!!! info Apache 2.0
+Allows users to use, distribute, modify, and distribute modified versions of the software for any purpose under the terms of the license, without worrying about license fees.
+
+## Advantages of OS-LLMs
+
+- Transparency
+- Fine-tuning
+- Community
+
+Open-source LLMs not only enable free usage but also impress with transparency by providing insight into architecture, functionality, and training data. Flexibility is enhanced through fine-tuning, allowing customization with proprietary data. Another advantage is the community, fostering continuous improvements and providing a space for the exchange of ideas.
+
+## Use Cases
+
+- Finance
+- Mathematics
+- Programming
+- Argumentation
+- Multilingual translation
+
+These language models can be trained for various use cases, with larger models aiming for a balanced mixture of these individual aspects.
+
+## 1) BLOOM
+
+<div style="text-align:center">
+  <img src="https://assets.website-files.com/6139f3cdcbbff3a68486761d/62cce3c835539c54f31329b1_image1.png" alt="Alt text" width="500"/>
+</div>
```

Comment:
No need for HTML `<div>` containers to embed images. You can use the Markdown syntax (also see the [assets info box](https://pkeilbach.github.io/htwg-practical-nlp/presentations/presentations/#submission-via-pull-request)

```suggestion
![Bloom Logo](https://assets.website-files.com/6139f3cdcbbff3a68486761d/62cce3c835539c54f31329b1_image1.png)
```

In case you need to adjust the width (but usually the original size works just fine, Mkdocs should adjust the image to the width in case its too big):

```
![Bloom Logo](https://assets.website-files.com/6139f3cdcbbff3a68486761d/62cce3c835539c54f31329b1_image1.png){ width=50% }
```

Please make sure to adjust the ["Alt text"](https://www.w3schools.com/tags/att_img_alt.asp) place holder.

Please adjust that throughout the document.